src/sentry/static/sentry/app/components/list.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import theme from 'app/utils/theme';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+export const List = styled('ul')`
+  list-style: none;
+  padding: 0;
+  margin-bottom: ${space(2)};
+
+  ol& {
```

Comment:
```suggestion
  ol & {
```
Should this have a space?

Reply:
Nope! It’s Emotion’s way of saying "when this styled component is an ol, do the following...”

Reply:
Neat. TIL.